docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -4174,6 +4174,48 @@ Port  Trunk  Security
   24         Enabled
 -----
 
+Pica8
+~~~~~
+
+PacketFence supports Pica8 switches without VoIP using CoA to:
+
+* bounce-host-port
+* reauthenticate-host
+
+Notes
+
+* **SNMP is not supported yet**
+* **Port Security is not supported**
+
+For interfaces with MAC Authentication, perform:
+
+  set interface gigabit-ethernet ge-1/1/25 family ethernet-switching port-mode trunk
+  set protocols dot1x interface ge-1/1/25 auth-mode mac-radius
+  set protocols dot1x interface ge-1/1/25 dynamic-vlan-enable true
+  set protocols dot1x traceoptions interface ge-1/1/25 flag all disable false
+
+For interfaces with 802.1X, perform:
+
+  set interface gigabit-ethernet ge-1/1/4 family ethernet-switching port-mode trunk
+  set protocols dot1x interface ge-1/1/4 dynamic-vlan-enable true
+  set protocols dot1x interface ge-1/1/4 dynamic-vlan-enable true
```

Comment:
duplicate line, please remove it

Reply:
Amjad Ali is the author of this pull request, he will work on all the review comments. Thank you all.